06_metrics.ipynb

```diff
@@ -0,0 +1,2393 @@
+{
```

Comment:
<p>Can you comment the first line out? Just to see if that is the culprit for the failures we are seeing?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Looking at the [Binderbot check results](https://github.com/xgcm/xgcm-examples/pull/18/checks), the error comes from rendering[01_ecco](https://github.com/xgcm/xgcm-examples/pull/18/checks#step:8:343) and [transform_mld_coordinates](https://github.com/xgcm/xgcm-examples/pull/18/checks#step:8:698)</p>